bin/docker/build

```diff
@@ -10,16 +10,23 @@ read -r version < ".ruby-version"
 DOCKER_TAG="byos_sinatra:latest"
 APP_ENV="production"
 BUILD_ARGS=("--build-arg" "RUBY_VERSION=${version}")
+
 while [ -n "${1-}" ]; do
-       if [ ${1-""} == "-t" ]; then
-              shift
-              DOCKER_TAG="${1:-$DOCKER_TAG}"
-       fi
-       if [ ${1-""} == "--dev" ]; then
-              APP_ENV="development"
-              BUILD_ARGS+=("--build-arg" "RACK_ENV=development" "--build-arg" "BUNDLE_WITHOUT=''")
-       fi
-       shift
+  case "${1-}" in
+    -t)
+      shift
+      DOCKER_TAG="${1:-$DOCKER_TAG}"
+      ;;
+    --dev)
```

Comment:
Based on our conversation above, I think you can simply this now. ...because we don't have to build for multiple environments, only production. Again, this is my fault for making this confusing but if we drop the notion of needing a dev build and only focus on production (leaving `.devcontainer` for local dev needs) then this simplifies your logic.